tests/unit/testplan/testing/multitest/driver/mykafka/test_kafka.py

```diff
@@ -39,11 +39,6 @@ def zookeeper_server():
 @pytest.fixture(scope="module")
 def kafka_server(zookeeper_server):
     server = kafka.KafkaStandalone("kafka", cfg_template=kafka_cfg_template)
-
```

Comment:
does this test pass?

Reply:
Also this test has a problem that it doesn't properly set runpath for this server